core/src/main/java/edu/wpi/grip/core/serialization/Project.java

```diff
@@ -151,7 +151,7 @@ public boolean trySave(File file) {
    * Save the project to a file.
    */
   public void save(File file) throws IOException {
-    try (final Writer writer = new OutputStreamWriter(new FileOutputStream(file),
+    try (Writer writer = new OutputStreamWriter(new FileOutputStream(file),
         StandardCharsets.UTF_8)) {
```

Comment:
??

Reply:
Redundant `final` modifier since objects declared in the try-with-resources block are implicitly final

Reply:
Okay.